src/Serializer/AbstractItemNormalizer.php

```diff
@@ -104,7 +104,7 @@ public function __construct(PropertyNameCollectionFactoryInterface $propertyName
      */
     public function supportsNormalization($data, $format = null)
     {
-        if (!\is_object($data)) {
+        if (!\is_object($data) || is_iterable($data)) {
```

Comment:
Isn't this a BC break? We already had problems with such checks IIRC.

Reply:
This is only to support the resource declaration on an Iterable what feature may this break?